packages/core/src/index.ts

```diff
@@ -463,6 +479,17 @@ class Hopr<Chain extends HoprCoreConnector> extends EventEmitter {
     return { receipt, status }
   }
 
+  public async getAcknowledgedTickets() {
+    return getAcknowledgedTickets(this)
+  }
+
+  public async submitAcknowledgedTicket(
+    ackTicket: Parameters<typeof submitAcknowledgedTicket>['1'],
```

Comment:
can we use the actual types here? This is a little hard to read.

Reply:
Fixed https://github.com/hoprnet/hoprnet/pull/583/commits/77a7f7fd232c5d6c85d03a8706136ac5be137a17#diff-9a4ceebe7c6f86856371906c3f061d3b56b7457022b05179884a113e7ced67e8R486